mmdet/datasets/transforms/formatting.py

```diff
@@ -112,6 +112,7 @@ def transform(self, results: dict) -> dict:
 
         img_meta = {}
         for key in self.meta_keys:
+            assert key in results, f'`{key}` is not found in `results`'
```

Comment:
Can also print the valid keys.
```suggestion
            assert key in results, f'`{key}` is not found in `results`, the valid keys are {list(results)}.'
```